cms/static/cms/js/require-config.js

```diff
@@ -91,7 +91,7 @@
             // end of Annotation tool files
 
             // externally hosted files
-            "mathjax": "//cdn.mathjax.org/mathjax/2.5-latest/MathJax.js?config=TeX-MML-AM_HTMLorMML-full&delayStartupUntil=configured", // jshint ignore:line
+            "mathjax": "//cdn.mathjax.org/mathjax/2.6-latest/MathJax.js?config=TeX-MML-AM_CHTML&delayStartupUntil=configured", // jshint ignore:line
```

Comment:
@adampalay  AFAIK, MML-full is still supported in 2.6.  This could be causing the display issue noted in your screenshots.  Chrome, FF, and Safari all support MathML now.  I'm assuming that because this config option was not included, MJ was not able to output MML and used HTML-CSS instead (which is very hacky, and not accessible).